Core/Core/Api/GraphQL/Models/ServerInfo.cs

```diff
@@ -0,0 +1,42 @@
+﻿#nullable disable
+using System;
+
+namespace Speckle.Core.Api.GraphQL.Models;
+
+public sealed class ServerInfo
+{
+  public string name { get; init; }
```

Comment:
All of the grahpql API stuff has been lowercase. If we want upper case, then we either need to add a lot of boilerplate to everything, e.g.
```cs
[JsonProperty(name = "name"]
public string name { get; init; }
```

or perhaps investigate serializer options for case insensitivity (unsure of the performance cost, and if this would work for objects that we serialize...)